src/tls.c

```diff
@@ -6163,8 +6163,16 @@ static int TLSX_Cookie_Parse(WOLFSSL* ssl, const byte* input, word16 length,
 
     /* client_hello */
     extension = TLSX_Find(ssl->extensions, TLSX_COOKIE);
-    if (extension == NULL)
-        return HRR_COOKIE_ERROR;
+    if (extension == NULL) {
+#ifdef WOLFSSL_DTLS13
+        if (ssl->options.dtls)
```

Comment:
Is this for dtls 1.3 only? I think you should add `IsAtLeastv13` to the check

Reply:
The cookie extension is only present in TLS 1.3 (I think) so you are right. I will add it.